states/MA/governor.yaml

```diff
@@ -0,0 +1,51 @@
+# Note that sometimes a feedback box pops up upon initial load of the page.  This doesn't show up every time, and doesn't appear twice in a row, so resubmissions will be successful if the feedback box is first encountered.
```

Comment:
Hehe on retries it'll be a fresh browser instance so... we'll just cross our fingers we don't get unlucky five times in a row =P

Reply:
Could we JS check for it's presence, then delete the enclosing element so that text entry works as planned?  This feels a bit bad like it could be a forever tech debt, but maybe not.  Well documented, I think it would be clear to people if this is ready to be culled someday.

Reply:
Hm I think adding JS to accommodate for this case *would* be tech debt =P  The way it is now seems fine.

Reply:
Maybe we could log an issue in the "nice to have" pile, and come back to it after the new forms are all done?

Reply:
@jason44406 hm one idea -- what if you do two identical `visit` directives back to back? Would that be robust? Might need a `wait` in between.

Reply:
@jason44406 Hm I just tried refreshing twice and then it disappeared -- maybe that would work reliably, three `visit` directives?!

Reply:
@jasonkb  -- I'm not sure, it seems random.  A dozen times on one browser and I couldn't get it to appear.  Fourth time on another and it appeared.

Reply:
Ah I see. So what you're saying is, 1st page load is as good as any. Sounds good.